test/node-unit/buffered-runner.spec.js

```diff
@@ -13,11 +13,10 @@ const BUFFERED_RUNNER_PATH = require.resolve(
 );
 const Suite = require('../../lib/suite');
 const Runner = require('../../lib/runner');
-const {createSandbox} = require('sinon');
+var sinon = require('sinon');
```

Comment:
How about `const sinon = require('sinon')`

Reply:
Missed changing this. Will update the PR.